testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -54,7 +54,11 @@ do_one_repo () {
   echo "Running script to create PR..."
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
-  local message="$(cat <<EOF
+  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
```

Comment:
Could you add a version of the PR description comment in the code? I had no idea what this was doing or used for without it.

Reply:
I don't understand. What should the comment be about?

Reply:
Ah I think I get it now. Was having trouble with sentence parsing

Reply:
Thanks!